docs/source/tutorials/token_classification/tutorial-p2.ipynb

```diff
@@ -0,0 +1,522 @@
+   "id": "1e6e2963",
+   "metadata": {},
+   "source": [
+    "Let's look at the top 20 examples cleanlab thinks are most likely to be incorrectly labeled. We obtain the sentences from the original file to display the word-level token label issues in context. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "639a34de",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# %%capture \n",
+    "# !wget https://data.deepai.org/conll2003.zip && mkdir data \n",
+    "# !unzip conll2003.zip -d data/ && rm conll2003.zip "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "f632872e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# collapse this \n",
+    "filepaths = ['data/train.txt', 'data/valid.txt', 'data/test.txt'] \n",
+    "entities = ['O', 'B-MISC', 'I-MISC', 'B-PER', 'I-PER', 'B-ORG', 'I-ORG', 'B-LOC', 'I-LOC']\n",
+    "entity_map = {entity: i for i, entity in enumerate(entities)} \n",
+    "\n",
+    "def readfile(filepath, sep=' '): \n",
+    "    lines = open(filepath)\n",
+    "    \n",
+    "    data, sentence, label = [], [], []\n",
+    "    for line in lines:\n",
+    "        if len(line) == 0 or line.startswith('-DOCSTART') or line[0] == '\\n':\n",
+    "            if len(sentence) > 0:\n",
+    "                data.append((sentence, label))\n",
+    "                sentence, label = [], []\n",
+    "            continue\n",
+    "        splits = line.split(sep) \n",
+    "        word = splits[0]\n",
+    "        if len(word) > 0 and word[0].isalpha() and word.isupper():\n",
+    "            word = word[0] + word[1:].lower()\n",
+    "        sentence.append(word)\n",
+    "        label.append(entity_map[splits[-1][:-1]])\n",
+    "\n",
+    "    if len(sentence) > 0:\n",
+    "        data.append((sentence, label))\n",
+    "        \n",
+    "    given_words = [d[0] for d in data] \n",
+    "    given_labels = [d[1] for d in data] \n",
+    "    \n",
+    "    return given_words, given_labels \n",
+    "\n",
+    "given_words, given_labels = [], [] \n",
+    "\n",
+    "for filepath in filepaths: \n",
+    "    words, label = readfile(filepath) \n",
+    "    given_words.extend(words) \n",
+    "    given_labels.extend(label)\n",
+    "    \n",
+    "sentences = list(map(get_sentence, given_words)) \n",
+    "\n",
+    "sentences, mask = filter_sentence(sentences) \n",
+    "given_words = [words for m, words in zip(mask, given_words) if m] \n",
+    "given_labels = [labels for m, labels in zip(mask, given_labels) if m] \n",
+    "\n",
+    "maps = [0, 1, 1, 2, 2, 3, 3, 4, 4] \n",
+    "given_labels = [mapping(labels, maps) for labels in given_labels] "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "65421a2d",
+   "metadata": {},
+   "source": [
+    "We show the top 20 potential label issues. Given that `O` and `MISC` are hard to distinguish and can sometimes be ambiguous, they are excluded from the examples below. "
```

Comment:
Explain how each entry tuple of `exclude` arg below works, and how the particular indices you chose to list there map to `O` and `MISC`